dashboard/modules/snapshot/snapshot_schema.json

```diff
@@ -102,6 +102,12 @@
                   "actorClass": {
                     "type": "string"
                   },
+                  "currentWorkerId": {
```

Comment:
This is a test case because the existing `test_snapshot` validates the schema

Reply:
👍